war/WEB-INF/applicationContext-security.xml

```diff
@@ -22,16 +22,16 @@
     	<csrf disabled="true"/>
         <custom-filter position="PRE_AUTH_FILTER" ref="preAuthnHarpFilter"/>
         <intercept-url pattern="/HarpSupport.html" access="permitAll()"/>
-        <intercept-url pattern="/HarpLogin.html" access="permitAll()"/>
+        <intercept-url pattern="/Login.html" access="permitAll()"/>

```

Comment:
Should we see legacylogin listed somewhere here? Or we just leave it for development only

Reply:
Development only. If it is made available in PROD, we'd be non-compliant.

Reply:
Thanks for the clarification